examples/guided.py

```diff
@@ -97,33 +121,45 @@ def ask_user_questions():
 
 					# Get a valid & supported filesystem for the partition:
 					while 1:
-						new_filesystem = input(f"Enter a valid filesystem for {partition} (leave blank for {partition.filesystem}): ").strip(' ')
+						new_filesystem = input(
+							f"Enter a valid filesystem for {partition} (leave blank for {partition.filesystem}): "
+						).strip(' ')
 						if len(new_filesystem) <= 0:
 							if partition.encrypted and partition.filesystem == 'crypto_LUKS':
 								old_password = archinstall.arguments.get('!encryption-password', None)
 								if not old_password:
 									old_password = input(f'Enter the old encryption password for {partition}: ')
 
-								if (autodetected_filesystem := partition.detect_inner_filesystem(old_password)):
+								if (
+									autodetected_filesystem := partition.detect_inner_filesystem(old_password)
+								) :
 									new_filesystem = autodetected_filesystem
 								else:
-									archinstall.log(f"Could not auto-detect the filesystem inside the encrypted volume.", fg='red')
-									archinstall.log(f"A filesystem must be defined for the unlocked encrypted partition.")
+									archinstall.log(
+										f"Could not auto-detect the filesystem inside the encrypted volume.", fg='red'
+									)
+									archinstall.log(
+										f"A filesystem must be defined for the unlocked encrypted partition."
+									)
 									continue
 							break
 
 						# Since the potentially new filesystem is new
 						# we have to check if we support it. We can do this by formatting /dev/null with the partitions filesystem.
 						# There's a nice wrapper for this on the partition object itself that supports a path-override during .format()
 						try:
-							partition.format(new_filesystem, path='/dev/null', log_formatting=False, allow_formatting=True)
+							partition.format(
+								new_filesystem, path='/dev/null', log_formatting=False, allow_formatting=True
+							)
 						except archinstall.UnknownFilesystemFormat:
-							archinstall.log(f"Selected filesystem is not supported yet. If you want archinstall to support '{new_filesystem}', please create a issue-ticket suggesting it on github at https://github.com/archlinux/archinstall/issues.")
+							archinstall.log(
+								f"Selected filesystem is not supported yet. If you want archinstall to support '{new_filesystem}', please create a issue-ticket suggesting it on github at https://github.com/archlinux/archinstall/issues."
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability.